src/data/markdown/translated-guides/en/01 Getting started/02 Installation.md

```diff
@@ -10,8 +10,8 @@ k6 has packages for Linux, Mac, and Windows. As alternatives, you can also use a
 ### Debian/Ubuntu
 
 ```bash
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
-echo "deb https://dl.k6.io/deb stable main" | sudo tee /etc/apt/sources.list.d/k6.list
+sudo gpg --no-default-keyring --keyring /usr/share/keyrings/k6-archive-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C5AD17C747E3415A3642D57D77C6C491D6AC1D69
```

Comment:
I wonder if we should suggest using `gpg2` here, in case someone actually has GPG v1 installed and this command doesn't work for them.

I tend to always use `gpg2` just in case, but this is probably fine.

Also, why name it `k6-archive-keyring.gpg`? Why "archive"? Maybe `k6.io-keyring.gpg`?

Sidenote: it's messy that users now have to interact with GPG directly. `yum`/`dnf` handles this in a much nicer way, and actually verifies package signatures. :bulb:

Reply:
wilco 👍🏻 

I'm gonna look into gpg2. It seems `gpg` is mapped to gpg version 2.2 as a default on my system. It's worth explicating it, though, indeed 👍🏻 

The `archive-keyring.gpg` suffix is the one that is already used by Ubuntu (and Debian) I believe. If you check out the content of `/usr/share/keyrings`, you'll see that `apt-key` already used this suffix all along 😸 

Sidenote: I must say, although the system indicates nicely that `apt-key` is indeed deprecated, it didn't make it any implicit what the "official" and "supported" alternative would be. I was glad some discussions, posts, and examples from Signal existed. In their defense, I haven't looked really hard for official docs, either. But I guess it's somewhat better for security to "force" users to do these steps explicitly? (Although it's somewhat more complex, and "complexity eats security for breakfast", and leaves room for mistakes...)

Reply:
`gpg` is probably fine; v2 should be the default on all systems by now.

Re: `archive`, I thought it was because Ubuntu's repo URL is http://archive.ubuntu.com/, but Debian also refers to it as "package archive" and has http://archive.debian.org/. This seems like a generic name for a package repository, so it's fine if we use the same nomenclature. :man_shrugging:

I think using `gpg` _is_ the "official" alternative now. I also didn't find any official docs explaining this, or _why_ it was deprecated to begin with, just some SO posts and hints on bugs.debian.org that people should stop using `apt-key`. Very poorly managed all around. :face_exhaling: We're OK if this is how Signal docs explain it, though.